src/anomalib/metrics/per_image/binclf_curve_numpy.py

```diff
@@ -180,7 +162,6 @@ def binclf_multiple_curves(
     scores_batch: ndarray,
```

Comment:
this function was like a demux for the two versions (python, numba), but without numba it becomes kind of redundant with the one that calls it (there are the validations tho)

Reply:
Yeah I plan to simplify this with several PRs